test/simpleTest.js

```diff
@@ -18,18 +18,29 @@ describe("simple", function() {
 	test("simple2", ".class { a: b c d; }\n.two {}", [
 		[1, ".class { a: b c d; }\n.two {}", ""]
 	]);
+	test("escape characters (uppercase)", ".class { content: \"\\F10C\" }", [
+		[1, ".class { content: \"\\F10C\" }", ""]
+	]);
+	// Need uncomment after resolve https://github.com/css-modules/postcss-modules-local-by-default/issues/108
```

Comment:
Not sure about these tbh, maybe better keep a note and add them when it is resolved :)